Module_4A/countries_data_handling.rb

```diff
@@ -0,0 +1,129 @@
+# ---------------- military module ---------------- 
+module MilitaryModule
+    attr_accessor :army_strength
+  
+    def initArmyStrength(army_strength)
+      @army_strength = army_strength
+    end
+  
+    def war_win_probability
+      return @army_strength > 300000
+    end
+  end
+  
+  
+  # ------------------------ loan_module ------------------------
+  module LoanModule
+    def loan_eligibility
+      # to determine whether the country is eligible for a loan from IMF or WB
+      return @gdp > 1000000
+    end
+  end
+  
+  # ----------------------- security_council_module -------------------------
+  module SecurityCouncilModule
+    def un_security_council_seat_eligibility
+      # to determine whether the country is eligible for a seat in the UN Security Council
+      return @army_strength > 500000
+    end
+  end
+  
+  
+  # ------------------------ country Class -----------------------------------
+  class Country
```

Comment:
Try splitting the big classes into seperate file

Reply:
Updated :+1: